### PR TITLE
new convention prefix change

### PR DESCRIPTION
**Pull request by ETLDevs**

### CODE REVIEW

File: src/pages/app/projects/index.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export default function ProjectsRoute({
   useEffect(() => {
     if (
       !projectType ||
-      Object.values(ProjectType).every((t) => t !== projectType)
+      Object.values(projectType).every((t) => t !== projectType)
```
Comment (shlomogcap) [inline src/pages/app/projects/index.tsx:28]: this should be EProjectType... this checks whether `projectType` is query param is one of know `EProjectType`

File: change-prefix.sh
Comment (shlomogcap) [inline change-prefix.sh:1]: this bash script should not be committed

File: src/lib/components/ProjectPage/NewProjectPage.tsx [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
   const { closeModal } = useModalContext();
   const addProjectText = PROJECT_DISPLAY_TEXTS.he.getAddNewText(projectType);
   const projectsTypeBreadCrumb: IBreadcrumbProps = {
     text: PROJECT_DISPLAY_TEXTS.he.projectTypes[projectType],
     id: 'projectType',
     navList: [
-      ProjectType.Residential,
-      ProjectType.Entrepreneurship,
-      ProjectType.PublicSpace,
+      projectType.Residential,
```
Comment (shlomogcap) [inline src/lib/components/ProjectPage/NewProjectPage.tsx:22]: this should be EProjectType ... 
we looping over the enum values (same as ObjectValues(EProjectType).map(....)) compare it with the current projectType value provided by props

File: src/lib/components/commons/FilterPanel/FilterPanel.styled.ts [hunk 3]
```diff
@@ -36,11 +36,11 @@ export const StyledFilterButton = styled(StyledButton)<IStyledFilterProps>`
     isButtonGroup &&
     css`
       border-radius: 0;
-      &:last-of-type {
+      &:last-of-type I{
```
Comment (shlomogcap) [inline src/lib/components/commons/FilterPanel/FilterPanel.styled.ts:39]: what is that?

File: src/lib/components/commons/FilterPanel/FilterPanel.styled.ts [hunk 4]
```diff
@@ -36,11 +36,11 @@ export const StyledFilterButton = styled(StyledButton)<IStyledFilterProps>`
...
         border-end-end-radius: 0.8rem;
         border-start-end-radius: 0.8rem;
       }
-      &:first-of-type {
+      &:first-of-type I{
```
Comment (shlomogcap) [inline src/lib/components/commons/FilterPanel/FilterPanel.styled.ts:43]: ?

File: src/pages/app/projects/index.tsx (hunk 1 above)
Comment (ETLDevs) [reply src/pages/app/projects/index.tsx:28]: fixed

File: src/lib/components/ProjectPage/NewProjectPage.tsx [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
+      projectType.Entrepreneurship,
```
Comment (ETLDevs) [inline src/lib/components/ProjectPage/NewProjectPage.tsx:23]: fixed!

File: src/lib/components/commons/FilterPanel/FilterPanel.styled.ts (hunk 3 above)
Comment (ETLDevs) [reply src/lib/components/commons/FilterPanel/FilterPanel.styled.ts:39]: lol its beacause of the change-prefix script!
it iditified 'type' and add capital I after...

File: change-prefix.sh
Comment (ETLDevs) [reply change-prefix.sh:1]: ok

**Review by shlomogcap — APPROVED** [review]

**Pull request merged**